### ISSUE TITLE
iPhone 6 iOS 12.4.6 Unsupported

### ISSUE THREAD

**Issue by IsmaeelAkram** [pwn20wndstuff/Undecimus#2024]
**Describe the bug**
I expected for the blue button to say Jailbreak instead of unsupported. I am using the latest version and it says iOS 11 to 13.5.

**To Reproduce**
Open unc0ver on iPhone 6 iOS 12.4.6.

**Screenshots**

**Device (please complete the following information):**
 - iOS Version: iOS 12.4.6
 - iOS Device: iPhone 6
 - unc0ver Version: 5.0.1

**Place an "x" between the brackets if true:**
 - [x] this is a bug others will be able to reproduce
 - [x] this issue is present with all tweaks uninstalled(except for default packages) or disabled
 - [x] this issue is present after a rootfs restore
 - [x] this issue is present on the latest version of unc0ver

**Logs**
If applicable, add logs or error messages here.

**Comment by Xuyiyang23333** [comment]
My system version is 12.4.7 and it is also showing unsupported.

**Comment by SarKaa** [comment]
Maybe it says that because your device is unsupported, what don't you understand, it says it's unsupported because it is, what do you expect the Devs to do

**Comment by IsmaeelAkram** [comment]
@SarKaa Unc0ver claims to be supported on all devices with iOS between 11 and 13.5. Stop trying to get joy out of putting down questions without even knowing anything.

**Comment by SarKaa** [comment]
> @SarKaa Unc0ver claims to be supported on all devices with iOS between 11 and 13.5. Stop trying to get joy out of putting down questions without even knowing anything.

Yeah but if you actually look around, you’ll find that unc0ver is definitely not supported on iOS 12.3 or iOS 12.4.1 to iOS 12.4.7

I’m not trying to get joy out of putting down questions, I actually read and don’t depend on the developer for everything

Next time, try reading peoples responses on reddit and stuff and then decide who’s the idiot

Your the one that wants to do something that simply isn’t possible currently, what do you want me to do, apart from tell you that it’s not possible, don’t blame me for you not knowing what your doing

I agree the unc0ver team said that they supported all signed firmware, but if you read the fine print, you would see that it didn’t mean all firmwares

Don’t say that I don’t know anything, your the one that hasn’t read the fine print, your the one that is asking to do the impossible (for now), and you blaming me for simply telling you that it’s impossible. It’s people like you that piss me off, you think you can be spoonfed, and when someone tells you that something can’t be done, you don’t understand how anything works and start asking for absurd things and blame others for your stupidity

**Comment by IsmaeelAkram** [comment]
I've definitely looked at Reddit and I've scoured the internet for answers but only select people were able to fix it. Some people were able to do it, and some weren't. This is the question. And if you knew it was totally impossible, then you couldn't have said that in more detail. Not "Maybe your device is unsupported."

**Comment by Xuyiyang23333** [comment]
I saw someone online saying that unc0ver supports 12.4.4, I don't know if it's true or not.

**Comment by SarKaa** [comment]
> I've definitely looked at Reddit and I've scoured the internet for answers but only select people were able to fix it. Some people were able to do it, and some weren't. This is the question. And if you knew it was totally impossible, then you couldn't have said that in more detail. Not "Maybe your device is unsupported."

Read the whole message, I literally said "it says it's unsupported because it is"
What more do you want, you could have read my whole response if that was the problem

**Comment by Ben569483** [comment]
It's because 12.4.7 is a new iPhoneOS for older phones and is not supported.

**Comment by IsmaeelAkram** [comment]
> > I've definitely looked at Reddit and I've scoured the internet for answers but only select people were able to fix it. Some people were able to do it, and some weren't. This is the question. And if you knew it was totally impossible, then you couldn't have said that in more detail. Not "Maybe your device is unsupported."
> 
> Read the whole message, I literally said "it says it's unsupported because it is" What more do you want, you could have read my whole response if that was the problem

ur a bum
